scripts/README.md

```diff
@@ -23,14 +23,15 @@ To build and install GenomicsDB using Docker, specify the following *optional* b
   | --- | --- |
   | os=ubuntu:trusty\|centos:7\|\<linux_base:ver\> | centos:7 |
   | user=<user_name> | genomicdb |
-  | branch=master\|develop\|<any_branch> | master |
```

Comment:
Yes, the dockerfile no longer uses the `branch` ARG. The idea behind removing it was that the Dockerfile and all the scripts it uses reside in our repository. So, it seemed natural to gives folks a way to build a docker image that uses that repository (and any changes they may make). 

Can you elaborate on your use case again?

Reply:
Basically, from GenomicsDB-Python, I create a centos6 image with all the Python distributions we care about. And then run something like `docker build --build-arg os=all_python:centos6 --build-arg distributable_jar=true --build-arg branch=ng_use_hdfs  -t genomicsdb:all_python .` to test things out. I usually just go to the GenomicsDB repository cloned on my machine. But, with this change I will have to specifically checkout the branch.

Reply:
Apologies because I don't fully understand the flow here, and what is being tested out exactly. If you're testing out GenomicsDB changes with GenomicsDB-Python, I presume you must already have that GenomicsDB branch somewhere...? What am I missing? Maybe a short meeting would help here.

Reply:
Yes, let's chat tomorrow. My branch was on the Mac and I was building on the Centos Box. For now, I just checked out the branch on Centos and am building from there.